src/AppInstallerCommonCore/Public/winget/ManifestCommon.h

```diff
@@ -130,12 +130,15 @@ bool HasExtension(std::string_view extension) const;
         AlreadyInstalled,
         Downgrade,
         BlockedByPolicy,
+        Custom

```

Comment:
>Custom [](http://example.com/codeflow?start=8&length=6)

nit: we usually end with , for easier merging

Reply:
added ,